src/server/controllers/FileCheckController.ts

```diff
@@ -50,14 +50,16 @@ export class FileCheckController implements FileCheckControllerInterface {
         if (hasVirus) {
           const user = req.session?.user as UserType
           logger.warn(
-            `Malicious link attempt: User ${user?.id} tried to upload ${file.name}`,
+            `Malicious file attempt: User ${user?.id} tried to upload ${file.name}`,
           )
           res.badRequest(jsonMessage('File is likely to be malicious.'))
           return
         }
       } catch (error) {
-        logger.error('Unable to check file: ', error)
-        res.serverError(jsonMessage(error.message))
+        logger.error('Unable to scan file: ', error)
+        res.badRequest(
+          jsonMessage('Your file could not be scanned by antivirus software.'),
+        )
```

Comment:
This should be more generic - FileCheckController is not meant to exclusively be for virus scanning but for all kinds of file checks, including, eg, the check for file type.

Reply:
Right, that's fine then.